RadarSDK/Radar.m

```diff
@@ -125,50 +125,69 @@ + (void)trackOnceWithDesiredAccuracy:(RadarTrackingOptionsDesiredAccuracy)desire
+                                         [RadarUtils runOnMainThread:^{
+                                             completionHandler(status, location, events, user);
+                                         }];
+                                     }
+                                 }];
                          };
 
                          if (beacons) {
-                             [[RadarAPIClient sharedInstance] searchBeaconsNear:location
-                                                                         radius:1000
-                                                                          limit:10
-                                                              completionHandler:^(RadarStatus status, NSDictionary *_Nullable res, NSArray<RadarBeacon *> *_Nullable beacons) {
-                                                                  if (status != RadarStatusSuccess || !beacons) {
-                                                                      callTrackAPI(nil);
-
-                                                                      return;
-                                                                  }
-
-                                                                  [[RadarLocationManager sharedInstance] replaceSyncedBeacons:beacons];
-
-                                                                  [RadarUtils runOnMainThread:^{
-                                                                      [[RadarBeaconManager sharedInstance]
-                                                                               rangeBeacons:beacons
-                                                                          completionHandler:^(RadarStatus status, NSArray<NSString *> *_Nullable nearbyBeacons) {
-                                                                              if (status != RadarStatusSuccess || !nearbyBeacons) {
-                                                                                  callTrackAPI(nil);
-
-                                                                                  return;
-                                                                              }
-
-                                                                              callTrackAPI(nearbyBeacons);
-                                                                          }];
-                                                                  }];
-                                                              }];
+                             [[RadarAPIClient sharedInstance]
+                                 searchBeaconsNear:location
+                                            radius:1000
+                                             limit:10
+                                 completionHandler:^(RadarStatus status, NSDictionary *_Nullable res, NSArray<RadarBeacon *> *_Nullable beacons,
+                                                     NSArray<NSString *> *_Nullable beaconUUIDs) {
```

Comment:
We should settle on a naming convention for this param at weekly today so we can test this; I'm using `projectBeaconUUIDs` and @kochis has thrown out `monitoredBeacons`

Reply:
Yeah, now that we are returning two different types of beacon ids (_nearby_ beacons, a _project monitored_ beacons), it would be good have a naming convention that highlights the difference between them.

My suggestion was `nearbyBeacons` and `monitoredBeacons`, but open to other ideas